plugins/zlux-plugins/action.yml

```diff
@@ -87,8 +90,34 @@ runs:
       run: |
         cd ../zlux/zlux-build 
         ant removeSource
+        cd ../$REPOSITORY_NAME/web
+        find . -type f -name '*.map*' -exec rm {} \;
```

Comment:
This will capture `.mappings`
try instead 2 commands:

find . -type f -name '*.map.*' -exec rm {} \;
find . -type f -name '*.map' -exec rm {} \;